### ISSUE TITLE
Missing "Skip to main content" 

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/resources-ressources#268]
**Describe the bug**
No link at the top of each page that goes directly to the main content area.

**To Reproduce**
Tab on keyboard when on the site.

**Expected behavior**
Best practice: First instance should be a "Skip to main content' or 'Skip navigation' link at the top of the page, visible when receiving focus.

**Impact**
Whereas sighted visitors may be able to use visual cues, visitors using screen readers may have to listen to a long list of links. Possible solution: Giving users the option to bypass links as they navigate the page .

**Screenshots**

**Desktop (please complete the following information):**
 - Chrome

**Additional context**
Found during manual a11y checks with SiteImprove.

**Comment by daine** [comment]
Grooming notes:
Adding WCAG link: https://www.w3.org/WAI/GL/WCAG20-TECHS/G1.html
